infrastructure/lambda/template.yaml

```diff
@@ -878,6 +883,25 @@ Resources:
         AttributeName: expiryDate
         Enabled: true
 
+KVBAnswerTable:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      TableName: !Sub "kvb-answer-${AWS::StackName}"
+      BillingMode: "PAY_PER_REQUEST"
+      PointInTimeRecoverySpecification:
+        PointInTimeRecoveryEnabled: true
+      AttributeDefinitions:
+        - AttributeName: "questionId"
+          AttributeType: "S"
+        - AttributeName: "answer"
+          AttributeType: "S"
+      KeySchema:
```

Comment:
The `KeySchema` refers to attributes that define the primary key.
Th primary key must be unique.
Therefore, it makes sense to add a new attribute e.g. `id` in the `AttributeDefinitions` because the same questionId will be written to the table for multiple items.
We can just write a random UUID value to the id field to ensure unique values

Reply:
This doesn't feel correct - we're going to end up with a new row every time someone goes through the service. We want a new row only when we see a new question/answer combo, don't we? Maybe I've misunderstood the purpose here.

Reply:
I suggested that we only write a single dynamo db item for each question id and then record the distinct answers in a separate field.
However, the team decided that the implementation should be simplified by just writing a question and answer combination every time. Irrespective of whether the answer has already been recorded.
To add a little context, these amends will only exist on a temporary/interim basis and will then be removed once the translations have been sorted for the dynamic answers etc.

Reply:
@galund Also, we don't care about data duplication at this point, we're just collecting information as simply as possible. We're collecting ~ 500 runs so that the data will be anonymous and so that we should have answers for all questions in our system.

When we create the query to export the data for analysis, we can de-dupe there.

As @tal-nagra said, once we have this data, we can remove this code.

Reply:
I feel like we might need something like this on an on-going basis, as you just never know when Experian are going to come up with more stuff?

(I have a slight worry about potentially being able to correlate answers with the times they were recorded, and therefore this is in effect a big pile of PII, but I can't quite see how you'd figure out the timings or the order of the rows here - so fine I guess.)

So sure, go for it, de-duplicate later: but put something in the team diary / on the JIRA board to 
(a) review whether this is still required or going to be needed longer-term
(b) check the costs of doing it this way vs row-per question (or answer)

Reply:
With regards, to your aforementioned concern, there is no timestamp being recorded therefore it isn't possible to know the time the dynamo db item was created.